src/modules/Post/index.jsx

```diff
@@ -115,108 +212,55 @@ export default class Post extends DefaultTemplate {
-              </header>
+            </header>
 
-              <div className="putainde-Post-md">
-                {
-                  (file.hero) &&
-                  <figure className="putainde-Post-hero">
-                    <img
-                      src={file.hero.src}
-                      alt={file.hero.alt || "Illustration"} />
-                    <small><a href={file.hero.credit}>Credit photo</a></small>
-                  </figure>
-                }
-                <div
-                  dangerouslySetInnerHTML={{__html: file.contents}}
-                />
-              </div>
+            <div className="r-Grid">
+              <div className="r-Grid-cell r-all--8of12 putainde-Post-contents">
+                <div className="putainde-Post-md">
+                  <div
+                    dangerouslySetInnerHTML={{__html: file.contents}}
+                  />
+                </div>
 
-              <footer className="putainde-Post-footer">
+                <footer className="putainde-Post-footer">
 
-                <div className="putainde-Post-footer-title">
-                  {i18n.pageActions}
-                </div>
-                <div className="r-Grid">
-                  <div className="r-Grid-cell r-all--1of3">
-                    <a
-                      className="putainde-Post-footer-action"
-                      href={
-                        Post.getGithubUrl(
-                          pkg.repositoryHttpUrl,
-                          "edit",
-                          filename
-                        )
-                      }
-                    >
-                      {i18n.pageEdit}
-                    </a>
-                  </div>
-                  <div className="r-Grid-cell r-all--1of3">
-                    <a
-                      className="putainde-Post-footer-action"
-                      href={
-                        Post.getGithubUrl(
-                          pkg.repositoryHttpUrl,
-                          "blame",
-                          filename
-                        )
-                      }
-                    >
-                      {i18n.pageBlame}
-                    </a>
-                  </div>
-                  <div className="r-Grid-cell r-all--1of3">
-                    <a
-                      className="putainde-Post-footer-action"
-                      href={
-                        Post.getGithubUrl(
-                          pkg.repositoryHttpUrl,
-                          "commits",
-                          filename
+                  {
+                    file.authors && file.authors.length === 1 &&
+                    <div>
+                      <Author author={file.authors[0]} isPost={true} />
+                    </div>
+                  }
+                  {
+                    file.authors && file.authors.length > 1 &&
+                    <div>
+                      <h3 className="putainde-WrittenBy">{i18n.writtenBy}</h3>
```

Comment:
Yeah. I think the entire CSS needs to be refactored a little bit. I am also thinking of adding postcss-bem-linter. What do you think ? https://github.com/necolas/postcss-bem-linter